### ISSUE TITLE
Outpainting always using the same image and mask, influencing outpaint result

### ISSUE THREAD

**Issue by sabashio** [AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin#288]
Even in new projects and outpainting on a completely white image, the image and mask previews always show up as the same picture from a past project and influence what is outpainted.  Is this information stored somewhere, cant seem to reset it?

![Capture](https://github.com/AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin/assets/130179115/485b623a-dfcd-4e92-a197-1b645297a40c)

**Comment by AbdullahAlfaraj** [comment]
No, that's not a normal behavior. Can you record a video so I can try replicating the bug?

**Comment by sabashio** [comment]
> 
I tried recording but the Auto-PS plugin isn't appearing in the video.  I am not doing much:

1. Open new project.
2. Draw 512x512 selection
3. Generate via txt2img
4. Apply desired generation.
5. Change to outpainting mode.
6. Select Latent Noise preset.
7. Make new selection, I try too keep a majority of the first generation in the new selection for context.
8. Press generate.
9. The image and mask previews appear but they are from previous, unsaved projects.  Outpainting results are made based one those older images and masks instead of the image in the selection.

NOTE:  This only seems to happen when work in a specific canvas dimensions.  This occurs when I use 1080x1920 but not in 1920x1080.  Will continue testing with other dimensions.

**Comment by sabashio** [comment]
> No, that's not a normal behavior. Can you record a video so I can try replicating the bug?

Did some more testing.  Seems like outpainting doesn't like 1080x1920 canvas size on my system.

I had 3 different tabs open and followed the same workflow as above.  Each tab had dimensions 1080x1920, 1920x1080, and 2100x1500 respectively.  The 2nd and 3rd tab were able to outpaint without issue.  The first tab seems to have tried to one of the other two tabs to determine its image and mask for outpainting.  The screenshot below shows all white, but tries showed images of the cat from the other tabs.  It seems to be dependent on the location of the selection I make for outpaiting and the location of the cat in the other tabs.  Similar to my initial report where it seems to have used a previous project for the image and mask but it chooses some arbitrary section of that older image.  In that case, no other tabs were open however.

Below are each tab had dimensions 1080x1920, 1920x1080, and 2100x1500 respectively:
![Screenshot (4)](https://github.com/AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin/assets/130179115/0282f1f4-c090-40e0-8f43-e45153c549e1)
![Screenshot (1)](https://github.com/AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin/assets/130179115/dfd7337b-718f-4ab5-84cc-6b70a10ec37d)
![Screenshot (2)](https://github.com/AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin/assets/130179115/47490612-cd27-462f-adae-b23d80aa7a44)

To sum up what I found,  outpainting on a 1080x1920 canvas will not use the image selected but will reference a different image from a past project to use as an image and mask.  Furthermore, the mask created is usually the same as the image (no black and white).  Subsequent testing did not reproduce this issue in other portrait orientations of different dimensions.
